js/graphdata.tsv

```diff
@@ -0,0 +1,356 @@
+datestamp	certsIssued	certsActive	fqdnsActive	regDomainsActive
+2015-09-12	1	NULL	NULL	NULL
```

Comment:
Is there a bug in your export script? It looks like the `fqdnsActive` and `regDomainsActive` columns are frequently null when they shouldn't be.